x-pack/plugins/apm/public/hooks/useTransactionBreakdown.ts

```diff
@@ -15,7 +15,7 @@ export function useTransactionBreakdown() {
     uiFilters,
   } = useUrlParams();
 
-  const { data = { timeseries: [] }, error, status } = useFetcher(
+  const { data = { timeseries: undefined }, error, status } = useFetcher(
```

Comment:
This looks redundant - or am I missing something?
```suggestion
  const { data = { timeseries }, error, status } = useFetcher(
```

Reply:
`timeseries` is not defined above for me to initialize the data with it, that's why I added the undefined. And it is initialized as `undefined` because I just show the `Loading chart component`, if the status is equal to `Loading` or `Pending` and the `timeseries` is undefined. Otherwise, I'd show the Loading every time the user pushes the `Refresh` button in the DatePicker or when the auto-refresh is on.

Reply:
> `timeseries` is not defined above for me to initialize the data with it, that's why I added the undefined.

I'm probably still missing your point but do we agree that `timeseries` will have `undefined` as value by default?

![image](https://user-images.githubusercontent.com/209966/98362578-c70bf980-202d-11eb-9129-da836c1ebd3e.png)

Reply:
I believe the example you gave is a bit different than what we have if I apply the change you suggested:
<img width="750" alt="Screenshot 2020-11-06 at 11 53 38" src="https://user-images.githubusercontent.com/55978943/98380018-c754c680-2026-11eb-9c51-761a774e8e19.png">

That's because I'm trying to initialize a value that has been restructured with `timeseries` which does not exists.